tests/tests-core/src/main/java/com/gentics/mesh/test/context/AbstractMeshTest.java

```diff
@@ -78,6 +78,7 @@ public MeshTestContext getTestContext() {
 	@Before
 	public void setupEventAsserter() {
 		eventAsserter = new EventAsserter(getTestContext());
+		testContext.waitAndClearSearchIdleEvents();
```

Comment:
The events, triggered by the initial data, have to be cleared, to prevent polluting the target event tracking.